week5/kaina/TAOBAO/index.css

```diff
@@ -0,0 +1,238 @@
+  z-index: 2;
+  padding-bottom: 5px;
+}
+
+.login-container img {
+  width: 200px;
+  height: 55px;
+  position: absolute;
+  top: 50px;
+  right: 5px;
+  z-index: 1;
+  margin: 0px;
+}
+
+.login-box {
+  border: 1px solid #e8e8e8;
+  border-top: 2px solid #ff4400;
+  box-shadow: 0px 2px 6px #acacac;
+  right: 0px;
+  width: 300px;
+  height: 350px;
+  margin: 40px 0px;
+  padding: 25px 25px 23px 25px;
+}
+
+.login-input-box {
+  display: flex;
+  position: relative;
+}
+
+.login-input-box input {
+  width: 250px;
+  height: 23px;
+  font-size: 14px;
+  padding: 9px 8px 8px 50px;
+  display: inline-block;
+  text-align: start;
+  border: 1px solid #ddd;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.login-input-box i {
+  width: 40px;
+  height: 40px;
+  background-color: #ddd;
+  display: block;
+  line-height: 40px;
+  color: #fff;
+  text-align: center;
+  position: absolute;
+  left: 1px;
+  top: 1px;
+  outline: 0;
+}
+
+.login-btn {
+  width: 300px;
+  height: 42px;
+  border: 0;
+  display: inline-block;
+  overflow: hidden;
+  vertical-align: middle;
+  line-height: 42px;
+  font-size: 16px;
+  font-weight: 700;
+  color: #fff;
+  background: #f40;
+  border-radius: 3px;
+  cursor: pointer;
+  zoom: 1;
+  letter-spacing: 2px;
+}
+
+.social-media-login {
+  background-color: #4c69ba;
+  border-color: #4c69ba;
+  border-radius: 2px;
+  color: #fff;
+  font-family: "Helvetica neue";
+  margin-top: 24px;
+  position: relative;
+  padding: 4px 5px 4px 16px;
+}
+
+.social-media-login i {
+  margin: 0px;
+  position: absolute;
+  left: 2px;
+}
+
+.login-footer-box {
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  margin-top: 25px;
+}
+
+.login-footer-box a {
+  color: #6c6c6c;
+  text-decoration: none;
+  margin-right: 10px;
+}
+
+footer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+footer a {
+  color: #6c6c6c;
+  text-decoration: none;
+}
+
+.help-container {
+  width: 100%;
+  border-top: 2px solid #f40;
+  border-bottom: 1px solid #e5e5e5;
+  background-color: #fafbfc;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+.help-box {
+  width: 300px;
+  height: 60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  border: 1px solid #e5e5e5;
+}
+
+.help-box img {
+  width: 40px;
+  margin-right: 10px;
+}
+
+.help-text h3 {
+  font-size: 14px;
+  color: #333;
+  margin-bottom: 0px;
+}
+
+.help-text p {
+  font-size: 12px;
+  color: #888;
+  width: 200px;
+  margin-top: 5px;
+  font-family: "Microsoft YaHei", tahoma, arial, "Hiragino Sans GB",
+    "\5b8b\4f53", sans-serif;
+}
+
+.help-footer-container {
+  margin-bottom: 15px;
+  max-width: 990px;
+  width: 100%;
+  font-size: 12px;
+  line-height: 30px;
+}
+
+.help-footer-web a {
+  padding: 0px 10px 0px 5px;
+  border-right: 1px solid #485065;
+  margin-left: -2px;
+}
+
+.help-footer-about a {
+  margin: 0px 2px;
```

Comment:
an inline element `<a>` does not have margin top or bottom, therefore you don't need to specify that the a tag has a 0 margin top and bottom. Come up with a more efficient way to style this a tag.

Reply:
inline block